makefiles/Makefile.devhub-subdomain-consolidation

```diff
@@ -0,0 +1,61 @@
+GIT_BRANCH=$(shell git rev-parse --abbrev-ref HEAD)
+USER=$(shell whoami)
+
+STAGING_URL="http://docs-devhub-staging.s3-website-us-east-1.amazonaws.com"
+STAGING_BUCKET=docs-devhub-staging
+
+STRAPI_URL=http://54.219.137.111:1337
+
+PRODUCTION_BUCKET=docs-devhub-prod
+PRODUCTION_URL=https://developer.mongodb.com
+
+PATH_PREFIX=developer
```

Comment:
We want the path prefix to simply be this. Gatsby is already set to support as such